src/interceptor/NodeClientRequest.ts

```diff
@@ -106,30 +106,30 @@ export class NodeClientRequest extends ClientRequest {
   emit(event: 'finish'): boolean;
   emit(event: 'pipe', src: Readable): boolean;
   emit(event: 'unpipe', src: Readable): boolean;
-  emit(event: string | symbol, ...args: any[]): boolean {
-    if (event === 'response') {
-      const response = args[0] as IncomingMessage;
-      const firstClone = cloneIncomingMessage(response);
-      const secondClone = cloneIncomingMessage(response);
-
-      async function emitResponse(
-        requestId: string,
-        message: IncomingMessage,
-        emitter: EventEmitter
-      ) {
-        const isomorphicResponse = await IsomorphicResponse.fromIncomingMessage(
-          message
-        );
-        emitter.emit('response', isomorphicResponse, requestId);
+  emit(event: string | symbol, ...args: any[]) {
+    if (event === 'response' && this.isInterceptable) {
```

Comment:
it is definitely makes sense to check `isInterceptable` in this place, so don't do any manipulations to the response